src/styles/blocks/wrapper.scss

```diff
@@ -0,0 +1,21 @@
+@import "../utils/variables";
+
+.wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($block-height, 1fr));
+  max-width: 1200px;
```

Comment:
No need to set max-width for the wrapper. Let's stretch the blocks so that they always fit the full screen.